### PR TITLE
Add security notice to file management tool

### PR DESCRIPTION
**Pull request by eyurtsev**
Add security notice to file management tool

### CODE REVIEW

File: libs/langchain/langchain/agents/agent_toolkits/file_management/toolkit.py [hunk 1]
```diff
@@ -29,7 +29,22 @@
 
 
 class FileManagementToolkit(BaseToolkit):
-    """Toolkit for interacting with a Local Files."""
+    """Toolkit for interacting with a Local Files.
```
Comment (obi1kenobi) [inline libs/langchain/langchain/agents/agent_toolkits/file_management/toolkit.py:32]:
```suggestion
    """Toolkit for interacting with local files.
```

File: libs/langchain/langchain/agents/agent_toolkits/file_management/toolkit.py [hunk 2]
```diff
@@ -29,7 +29,22 @@
...
+
+    *Security Notice*: This toolkit provides methods to interact with local files.
+        If providing this toolkit to an agent on an LLM, ensure to scope
```
Comment (obi1kenobi) [inline libs/langchain/langchain/agents/agent_toolkits/file_management/toolkit.py:35]:
```suggestion
        If providing this toolkit to an agent on an LLM, ensure you scope
```

File: libs/langchain/langchain/agents/agent_toolkits/file_management/toolkit.py [hunk 3]
```diff
@@ -29,7 +29,22 @@
...
+        to perform the desired operations.
+
+        By **default** the agent will have access to all files within
+        the root dir and will be able to Copy, Delete, Move, Read, Write
+        and List files in that directory.
+
+        Consider limiting access to particular directories (using root_dir)
+        and limiting the tools available to the agent to the file operations
+        that are necessary for the agent's intended use. In addition,
+        consider sandboxing the agent by running it in a container.
```
Comment (obi1kenobi) [inline libs/langchain/langchain/agents/agent_toolkits/file_management/toolkit.py:46]: Consider switching to a bullet point list for clarity. I'm going to add another recommendation as well:
```suggestion
        Consider the following:
        - Limit access to particular directories using `root_dir`.
        - Use filesystem permissions to restrict access and permissions to only
          the files and directories required by the agent.
        - Limit the tools available to the agent to only the file operations
          necessary for the agent's intended use.
        - Sandbox the agent by running it in a container.
```

**Review by obi1kenobi — APPROVED** [review]
Already better than status quo. Some suggestions for further improvement.

**Review by obi1kenobi — APPROVED** [review]

**Review by baskaryan — APPROVED** [review]

**Pull request merged**